src/scaffold/dna.rs

```diff
@@ -216,7 +216,8 @@ pub fn scaffold_dna(
                 properties: None,
                 quantum_time: None,
             },
-            version: None,
+            installed_hash: None,
+            _version: None,
```

Comment:
Shouldn't need it

```suggestion
```

Reply:
Closing this PR as it's branched off of main, but I have a local branch with the same changes branched off of `holochain-0.1` and I'll your suggested change there.